client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -24,11 +24,11 @@ class QueryContactDetailsCache extends Component {
 }
 
 QueryContactDetailsCache.propTypes = {
-	requesting: PropTypes.bool,
-	requestContactDetailsCache: PropTypes.func
+	isRequesting: PropTypes.bool.isRequired,
```

Comment:
Strictly speaking `isRequesting ` is not required, but I'm happy to leave it this way if you feel it's appropriate.